assets/js/characterCounter.js

```diff
@@ -0,0 +1,55 @@
+const inputParentElem = document.querySelector(
+    '[data-module="govuk-character-count"]'
+);
+
+if (inputParentElem) {
+    prepareCharCounter();
+}
+
+function prepareCharCounter() {
+    const inputElem = inputParentElem.querySelector(
+        '.govuk-js-character-count'
+    );
+    const commonParams = {
+        charLimit: Number(inputParentElem.dataset.maxlength),
+        inputElem,
+        inputHint: inputParentElem.querySelector(
+            '.govuk-character-count__message'
+        ),
+    };
+
+    displayInitialHintText(commonParams);
+    inputElem.addEventListener('input', (_event) =>
+        updateHintText(commonParams)
+    );
+}
+
+function displayInitialHintText(commonParams) {
+    const { charLimit, inputElem, inputHint } = commonParams;
+    const inputHasValue = inputElem.value.length > 0;
+
+    inputHasValue
+        ? updateHintText(commonParams)
+        : (inputHint.textContent = `You have ${charLimit} characters remaining`);
```

Comment:
you will need to check if there are polyfills on this project or some sort of transpilation (like babel) on these js files. 

I'm pretty sure template strings and destructure isn't IE11 compatible.. and yes, unfortunately we will have to support IE11.. it's refreshed every 6 months.. so hopefully we don't have to support it for much longer. Worth bookmarking this link! https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices